server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/reader/CobolLineReaderImpl.java

```diff
@@ -162,7 +173,7 @@ private SyntaxError createError(
                     .uri(uri)
                     .range(
                         new Range(new Position(lineNumber, start), new Position(lineNumber, stop)))
-                    .recognizer(getClass())
+                    .recognizer(MethodHandles.lookup().lookupClass())
```

Comment:
Why?

Reply:
Don't remember this change.  Definitely not required, will get rid of this.